test/src/test/java/hudson/tasks/ArtifactArchiverTest.java

```diff
@@ -39,12 +39,20 @@
 import java.net.HttpURLConnection;
 import java.util.Collections;
 import java.util.List;
+
+import hudson.util.FormValidation;
 import jenkins.util.VirtualFile;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertTrue;
 import static org.junit.Assert.*;
 import static org.junit.Assume.*;
 
+import org.junit.Assert;
 import org.junit.Rule;
 import org.junit.Test;
+
+import org.jvnet.hudson.test.Bug;
```

Comment:
Seems he added `@Bug` and then realised that it deprecated for `@Issue` that already imported.